### PR TITLE
Form template

### PR DESCRIPTION
**Pull request by sophyo**
added a basic layout and template styling, as a starting point for developing a general admin ui

### CODE REVIEW

File: erudit/editor/forms.py [hunk 1]
```diff
@@ -42,9 +42,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.helper = FormHelper()
-        self.helper.form_class = 'form-horizontal'
-        self.helper.label_class = 'col-lg-2'
-        self.helper.field_class = 'col-lg-8'
+        self.helper.form_class = 'form-vertical'
+        # self.helper.label_class = 'col-xs-12'
```
Comment (cormier) [inline erudit/editor/forms.py]: If there is no reason to keep it, do not hesitate to delete it.

File: erudit/erudit/static/sass/base/_typography.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/**
+ * Basic typography style for copy text
+ */
+
+body {
+	background-color: $colour-faint-grey * 1.08;
```
Comment (cormier) [inline erudit/erudit/static/sass/base/_typography.scss]: Are we mixing tabs and spaces?

**Comment by cormier** [comment]
Before merging this I think we should:
* Adopt a standard for indenting the scss files (tabs? 2 spaces? 4 spaces?) and use it everywhere
* Delete the commented code in forms.py if we're not going to use it
* Add a quick mention to the "sub readmes" in the UI development section of the principal `README.md` file

File: erudit/erudit/static/sass/base/_typography.scss (hunk 2 above)
Comment (sophyo) [inline erudit/erudit/static/sass/base/_typography.scss]: oops; this came from SASS I had previously written

**Comment by sophyo** [comment]
Thanks! Done and committed.

**Pull request merged**